contracts/factories/DelegatedManagerFactory.sol

```diff
@@ -234,23 +241,27 @@ contract DelegatedManagerFactory {
         require(msg.sender == initializeState[_setToken].deployer, "Only deployer can initialize manager");
         _initializeTargets.validatePairsWithArray(_initializeBytecode);
 
-        IDelegatedManager manager = initializeState[_setToken].manager;
```

Comment:
moving the new require statement above the owner fee updates

Reply:
+1